features/posix_spawn.c

```diff
@@ -23,6 +24,10 @@ pid_t __fork(void) {
     return -1;
 }
 
+// Exits with the following status codes:
+//    0 : fully functional
+//    1 : partly functional
+// >= 2 : unusable
```

Comment:
The only place that tests the `_lib_posix_spawn` symbol only distinguishes between two and everything else. In other words  we don't need to distinguish between partly functional and unusable. So this can be simplified to zero and non-zero.